harness/evidence/mutate.sh

```
@@ -0,0 +1,394 @@
+# 変異テスト: 証拠 digest まわりの各ゲートをわざと壊し、対応する test が落ちることを確かめる。
+#
+# 使い方: bash harness/evidence/mutate.sh
+# 出力の各行は「M 番号 + ラベル」と、その変異を入れたときの fail 件数。
+# **fail 0 の行は生存した変異**（そのゲートを壊しても test が落ちない = 検証が効いていない）。
+#
+# 実行件数も必ず突き合わせる。アンカーが実装の変更で外れると `assert count == 1` が落ちて
+# `&&` が短絡し、その変異は出力に何も出ないまま黙って飛ばされる。
+#
+# 中断すると変異が残る。その場合は `git checkout harness/`。
+set -u
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)